src/lib/jwk-store.ts

```diff
@@ -61,7 +61,11 @@ export class JWKStore {
    * @returns {Promise<JWK.Key>} The promise for the added key.
    */
   async add(jwk: JWK.Key): Promise<JWK.Key> {
-    const jwkUse: JWK.Key = { ...jwk, use: 'sig' };
+    const jwkUse: JWK.Key = { ...jwk };
+
+    if (!('use' in jwkUse)) {
+      Object.assign(jwkUse, { use: 'sig' });
```

Comment:
@poveden This work arounds TypeScript warning us against the `{ use: 'sig', ...jwk };` usage. Indeed, "use" is not optional in the `JWK.Key` type definition.

![image](https://user-images.githubusercontent.com/92363/98858294-aa722600-2460-11eb-8377-15597b3522be.png)